site/content/docs/user/quick-start.md

```diff
@@ -226,6 +226,18 @@ nodes:
 - role: worker
 ```
 
+### Configure kind to use a proxy
+If you are running kind in an environment that requires a proxy to connect outside you may need to use it.
+
+You can configure kind to use a proxy using the environment variables (uppercase takes precedence):
+
+* HTTP_PROXY or http_proxy
+* HTTPS_PROXY or https_proxy
+* NO_PROXY or no_proxy
```

Comment:
It would probably be helpful to link to some sort of reference regarding what these usually mean and what their values look like. perhaps docker's docs.

Reply:
maybe we can link to https://docs.docker.com/network/proxy/